.changeset/dull-words-work.md

```diff
@@ -0,0 +1,9 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+---
+---
+"section": tree
+---
+
```

Comment:
Can you add a short description of the change here? It only needs to be a short phrase and doesn't need punctuation at the end.

Reply:
The first sentence of the block below would work fine.